main/src/library/Core/Io/OutputStream.flix

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2020 Rasmus Larsen
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+use Core/Io/File.File;
+use Core/Io/IOError.IOError;
+
+namespace Core/Io/OutputStream {
+    pub opaque type OutputStream = ##java.io.OutputStream
+
+    pub def open(f: File): Result[OutputStream, IOError] & Impure = {
```

Comment:
Add documentation.

Also, both for InputStream and OutputStream, let us name the function simply `new`. Then you can write `InputStream.new(...)` which seems pretty nice and is used in other places.

Reply:
Damn! I really thought I'd gotten all of them.

I'll rename the methods.